byon-service/db_interface.py

```diff
@@ -0,0 +1,49 @@
+########
+# Copyright (c) 2015 GigaSpaces Technologies Ltd. All rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# * See the License for the specific language governing permissions and
+# * limitations under the License.
+import abc
+
+
+class DBinterface(object):
+    __metaclass__ = abc.ABCMeta
+
+    @abc.abstractmethod
+    def set_default_auth(self, auth):
+        """Set default auth(dictionary) with
+           credentials(username and password or keyfile) set in config"""
+
+    @abc.abstractmethod
+    def get_servers(self):
+        """Return all servers in database"""
+
+    @abc.abstractmethod
+    def add_server(self, server):
```

Comment:
why is this needed? for now our pool is static and all servers are loaded at initialization time.
i'm sure we will need this functionality later on, just trying to keep it as simple as possible for now.